### PR TITLE
Release candidate 0.1.4

### PR DESCRIPTION
**Pull request by AndyTitu**

### CODE REVIEW

File: client/release/RELEASE-NOTES [hunk 1]
```diff
@@ -1,5 +1,3 @@
 The v0.1.3 release of the Javascript SDK brings:
-* Support for item websites. You can now create, get, and edit websites for autofill within your 1Password items using item CRUD functions.
-
-
+* Support for validating secret references. You can now check that a secret reference is formatted correctly without having to resolve it or even authenticate, using the 'ValidateSecretReference' function.
```
Comment (MOmarMiraj) [inline client/release/RELEASE-NOTES:2]: Do we want to add something for the removal of return type for void functions?

As you can see any function that returned a `Promise<void>` we made it return nothing.

File: client/release/RELEASE-NOTES [hunk 2]
```diff
@@ -1,5 +1,3 @@
 The v0.1.3 release of the Javascript SDK brings:
```
Comment (MOmarMiraj) [inline client/release/RELEASE-NOTES:1]: 0.1.4

File: client/release/RELEASE-NOTES (hunk 1 above)
Comment (AndyTitu) [reply client/release/RELEASE-NOTES:2]: This is an internal simplification that doesn't modify the user experience in any way, so we shouldn't mention it.

File: client/release/RELEASE-NOTES (hunk 2 above)
Comment (AndyTitu) [reply client/release/RELEASE-NOTES]: Done, thanks!

File: examples/index.cjs [hunk 3]
```diff
@@ -114,5 +123,7 @@ async function manageItems() {
   await client.items.delete(item.vaultId, item.id);
 }
 
+
+
```
Comment (MOmarMiraj) [inline examples/index.cjs:127]: We can remove this extra diff.

**Review by MOmarMiraj — APPROVED** [review]

**Pull request merged**